vm/actor/src/code.rs

```diff
@@ -2,10 +2,11 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use cid::Cid;
+use serde::{Deserialize, Serialize};
 
 /// CodeID is the reference to the code which is attached to the Actor state.
 /// There are builtin IDs and the option for custom code with a Cid
-#[derive(PartialEq, Eq, Clone, Debug)]
+#[derive(PartialEq, Eq, Clone, Debug, Serialize, Deserialize)]
```

Comment:
I don't care, but just to note: enum default serialization is not very efficient so if the codeID is going to be serialized and deserialized a lot, this should probably be avoided.